www/js/lib/uiUtil.js

```diff
@@ -53,11 +53,33 @@ define([], function() {
         }
     }
 
+    function makeReturnLink(title) {
+        //Abbreviate title if necessary
+        var shortTitle = title.substring(0, 25);
+        shortTitle = shortTitle == title ? shortTitle : shortTitle + "..."; 
+        var link = '<h4><a href="#" onclick="$(\'#configuration\').hide();$(\'#formArticleSearch\').show();' +
+            '$(\'#articleContent\').show();$(\'#liHomeNav\').attr(\'class\',\'active\');$(\'#liConfigureNav\').attr(\'class\', \'\');' +
+            '$(\'#liAboutNav\').attr(\'class\', \'\');">&lt;&lt; Return to ' + shortTitle + '</a></h4>';
```

Comment:
I think we should avoid putting some HTML inside javascript.
This HTML code could probably be put in the HTML file, with the dynamic part identified by an id and filled from the javascript. Plus using the visibility to show/hide it.